src/python/pants/core/goals/fmt.py

```diff
@@ -279,7 +279,7 @@ async def fmt(
 
     for formatter, results in sorted(formatter_to_results.items()):
         if any(result.did_change for result in results):
-            sigil = console.red("𐄂")
+            sigil = console.yellow("𐄂")
```

Comment:
Since this isn't an error, this could maybe use a different symbol... `~` or something?